app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -339,47 +339,50 @@ fun CollapsibleHikesList(hikingRoutesViewModel: ListOfHikeRoutesViewModel, isSea
   val routes = hikingRoutesViewModel.hikeRoutes.collectAsState()
   val context = LocalContext.current
 
+  // Map of route ID to the elevation data of the route
+  val elevationDatas = remember { mutableStateOf(emptyMap<String, List<Double>>()) }
```

Comment:
Nitpick: Datas is not a word, maybe a different name like `elevationDataMappings` works too?